data-oriented-programming/model-optionality/ballerina/main.bal

```diff
@@ -0,0 +1,31 @@
+import ballerina/io;
+
+type Person record {|
+    int id;
+    string name;
+    //optional typed field
```

Comment:
I follow the following format when writing single-line code comments. 

`// Optional field`
- There exists a space right after `//` characters.
- Start the comment with an uppercase character.

I would like to see consistent format in code comments.

Reply:
Updated in a09d2a31e528e08ec9d858e59b89c4f0f4f7ae4e